src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -2542,7 +2554,6 @@ ClientChannel::LoadBalancedCall::LoadBalancedCall(
       call_start_time_(args.start_time),
```

Comment:
Although this is a primitive, I believe it's UB since arg is `move`d on line 2425.

Reply:
I don't understand this comment.  The `args` that's being moved on line 2425 is a local variable in the `ClientChannel::CallData::CreateDynamicCall()` method.  The `args` that's being used here is a parameter being passed into the `ClientChannel::LoadBalancedCall::LoadBalancedCall()` ctor.  I think these are completely different objects.  Am I missing something here?

Reply:
Oh I'm sorry. I somehow thought this a continuation. Please disregard.